toolprovider/mise/mise.go

```diff
@@ -21,15 +23,29 @@ import (
 // 3. Verify checksums
 // 4. Update version string and checksums below
 // 5. IMPORTANT, DO NOT FORGET: Mirror artifacts to GCS bucket (see bootstrap.go) in case github.com goes down
-const MiseVersion = "v2025.8.7"
+const miseVersion = "v2025.10.8"
 
-var MiseChecksums = map[string]string{
+var miseChecksums = map[string]string{
```

Comment:
How about `misePreviewChecksums`? Just to make it obvious this is not the only var to update